packages/skeleton-react/src/components/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,179 @@
+import { describe, expect, it } from "vitest";
+import { render, waitFor } from "@testing-library/react";
+
+import { Accordion, AccordionContext } from "./Accordion";
+
+// Accordion ---
+
```

Comment:
As with the App.test.tsx file I recommend replacing `toBeTruthy()` with more specific tests

Reply:
See: https://github.com/skeletonlabs/skeleton/pull/2566#discussion_r1539983435